src/runtime/components/SelectMenu.vue

```diff
@@ -281,6 +302,7 @@ function onUpdateOpen(value: boolean) {
     :disabled="disabled"
     :multiple="multiple"
     :by="by"
+    :ignore-filter="props.filter === false"
```

Comment:
Why not keep using `ignore-filter`? It's inherited from Combobox already and `filter` prop has been removed already.

Reply:
OK I'll remove it - didn't spot that!